### DISCUSSION TITLE
Unable to continue installation

### DISCUSSION THREAD

**Discussion by seasonlei** [MystenLabs/sui#10862] [Q&A]
cargo install --git https://github.com/MystenLabs/sui.git --branch devnet sui

![571A04B4-C2EB-4f34-9F48-9090723FFDA0](https://user-images.githubusercontent.com/60209171/231700416-fb6ceb2f-ee28-4bb5-9e5c-7d7a005d227a.png)

It's stuck in here

**Comment by threewebcode** [comment]
You can kill it and rebuild it.

**Comment by threewebcode** [comment]
Most of time,  your machine is not good enough as you know rust is expensive
